linera-views/src/dynamo_db.rs

```diff
@@ -193,6 +189,57 @@ fn extract_key_value_owned(
     Ok((key, value))
 }
 
+fn build_delete_transact(
+    key: Vec<u8>,
+    db: &DynamoDbClientInternal,
+) -> Result<TransactWriteItem, DynamoDbContextError> {
+    if key.is_empty() {
+        return Err(DynamoDbContextError::ZeroLengthKey);
+    }
+    let request = Delete::builder()
+        .table_name(&db.table.0)
+        .set_key(Some(build_key(key)))
+        .build();
+    Ok(TransactWriteItem::builder().delete(request).build())
+}
+
+fn build_put_transact(
+    key: Vec<u8>,
+    value: Vec<u8>,
+    db: &DynamoDbClientInternal,
+) -> Result<TransactWriteItem, DynamoDbContextError> {
+    if key.is_empty() {
+        return Err(DynamoDbContextError::ZeroLengthKey);
+    }
```

Comment:
Is there a limit for the key length, too? Should we check that?

Reply:
Yes, good point, added.